doc/examples/gestures/lib/main_mouse_movement.dart

```diff
@@ -48,10 +48,10 @@ class MyGame extends BaseGame with MouseMovementDetector {
   void update(double dt) {
     super.update(dt);
     if (target != null) {
-      _onTarget = _toRect().contains(target.toOffset());
+      _onTarget = _toRect().contains(target!.toOffset());
```

Comment:
Off topic: Dart could be smart enough to know that unwrap isn't necessary here :(

Reply:
it can't because it's a field, so it could have changed between lines. we need ifs with declaration :(